server/app/controllers/UserController.ts

```diff
@@ -63,13 +63,50 @@ export class UserController extends BaseHttpController {
         const accessToken = generateToken(user);
         response.jwt = accessToken;
       }
-
       return this.json(response);
     } catch (err) {
       return this.json(err.message, 400);
     }
   }
 
+  @Doc('Forgot password')
+  /**
+   * @desc        Send email to existing user
+   * @route       POST /user/forgot
+   * @access      Public
+   * @param req
+   * @returns     User JSON Format + Jwt token
+   */
+  @httpPost('/forgot')
+  public async forgot(req: Request): Promise<results.JsonResult> {
+    try {
+      const promise = await this.userService.forgotPassword(req.body);
+      return this.json(promise, 200);
+    } catch (err) {
+      return this.json(err.message, 300);
```

Comment:
Why 300?

Reply:
@Mackbellemore Because if 400, the console will show its an error, while I wish to hide it for the user and still be catch as an error. I am not sure what is the best practice in here.

Reply:
ah interesting, ya ive never had to do anything like that so i guess this is fine